src/js/stores/history.js

```diff
@@ -638,6 +647,21 @@ define(function (require, exports, module) {
         },
 
         /**
+         * New and improved "unified" history event handler.  payload.history determines
+         * whether or not history state is created or amended.
+         *
+         * @param {object} payload
+         */
+        _handleUnifiedHistory: function (payload) {
+            if (payload.history && payload.history.newState) {
+                log.debug("UnifiedHistory: Creating new history state");
```

Comment:
I don't want to commit more debug-only log statements like this unless you really, really think this will be useful later. If so, can I'd like to group them all somehow under a "history" tag kind of like CC Libraries does.